lib/decision_review_v1/utilities/form_4142_processor.rb

```diff
@@ -16,27 +16,27 @@ class Form4142Processor
 
       def initialize(form_data:, submission_id: nil)
         @form = form_data
+        @submission = Form526Submission.find_by(id: submission_id)
         @pdf_path = generate_stamp_pdf
         @uuid = SecureRandom.uuid
         @request_body = {
           'document' => to_faraday_upload,
           'metadata' => generate_metadata
         }
-        @submission = Form526Submission.find_by(id: submission_id)
       end
 
       def generate_stamp_pdf
         pdf = PdfFill::Filler.fill_ancillary_form(
           @form, @uuid, FORM_ID
         )
+        submission_date = @submission&.created_at&.in_time_zone('Central Time (US & Canada)')
         stamped_path = CentralMail::DatestampPdf.new(pdf).run(text: 'VA.gov', x: 5, y: 5,
-                                                              timestamp: @submission&.created_at)
+                                                              timestamp: submission_date)
         CentralMail::DatestampPdf.new(stamped_path).run(
           text: 'VA.gov Submission',
           x: 510,
           y: 775,
-          text_only: true,
-          timestamp: @submission&.created_at
+          text_only: true
```

Comment:
I had added this timestamp on L39 in my previous datestamp PR, but the `text_only` attribute means that the watermark should just include the text without a date, so I removed the timestamp here since it wouldn't be rendered anyways.

Reply:
Correct, this second call to `CentralMail::DatestampPdf` only functions to stamp "VA.gov Submission" at the top of each page